setup.py

```diff
@@ -54,9 +56,9 @@ def version():
     },
     install_requires=(
         'azure-identity>=1.7.1,<1.8.0',
-        'azure-mgmt-dns>=8.0.0,<8.1.0',
+        'azure-mgmt-dns>=8.1.0b1,<8.2.0',
```

Comment:
Had to pull in azure-mgmt-dns beta version to prevent regression in https://github.com/Azure/azure-sdk-for-python/issues/30686. Calling out here as an outstanding item to resolve before merging.